docs/reference/search/search-your-data/semantic-search-semantic-text.asciidoc

```diff
@@ -91,6 +89,9 @@ Every time you ingest data into the related `semantic_text` field, this endpoint
 <4> The field to store the text reindexed from a source index in the  <<semantic-text-reindex-data,Reindex the data>> step.
 <5> The textual data stored in the `content` field will be copied to `semantic_text` and processed by the {infer} endpoint.
 The `semantic_text` field will store the embeddings generated based on the input data.
+Copying the `content` field content to the `semantic_text` field is not required for using `semantic_text`.
+However, it is recommended because it makes it possible to perform hybrid search during which you perform lexical search on the text field and semantic search on the semantic_text field.
```

Comment:
```suggestion
However, it is recommended because it makes it possible to perform hybrid search. Hybrid search involves combining lexical search on the `text` field and semantic search on the `semantic_text` field.
```

🚗 :)